instruments/tests/test_tektronix/test_tekawg2000.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Unit tests for the Tektronix AGG2000 arbitrary wave generators.
+"""
+
+# IMPORTS #####################################################################
+
+from hypothesis import (
+    given,
+    strategies as st,
+)
+import numpy as np
+import pytest
+
+import instruments.units as u
+import instruments as ik
+from instruments.tests import expected_protocol, make_name_test
+
+
+# TESTS #######################################################################
+
+# pylint: disable=protected-access
+
+
+test_tekawg2000_name = make_name_test(ik.tektronix.TekAWG2000)
+
+
+# CHANNEL #
+
+
+@given(channel=st.sampled_from(range(2)))
```

Comment:
Hmmm, the only thing I wonder with these `st.sampled_from(range(2))` is if we are accidentally over-testing through repeated sampling of a small set. I'll want to give these a test locally to verify just to make sure.

Reply:
You're probably right. Since there are only two channels, sampling from `range(2)` is probably overkill. I can easily switch them back to simply "sampling" by hand, as done before.

Reply:
Yeah, when I first saw this usage recently I wondered "is this trying to use hypothesis for what could normally be done with pytest.parametrize" (i.e. generate combinations/permutations of parameter values). Further down there were combinations of float parameter ranges, where that approach probably makes more sense, as you don't have to pick out specific values as you would with pytest.